apps/wing/src/cli.ts

```diff
@@ -169,7 +169,7 @@ async function main() {
     .argument("[entrypoint]", "program .w entrypoint")
     .option(
       "-t, --platform <platform> --platform <platform>",
-      "Target platform provider (builtin: sim, tf-aws, tf-azure, tf-gcp, awscdk)",
```

Comment:
We should figure out how we improve the discoverability of non built in platforms?  

Maybe we can add here a link to a doc page with a list as a simple first step?

@boyney123 what do you think?

Reply:
Yeah I like the idea, maybe we can add a `List of platforms` page to the new platforms section.

<img width="249" alt="image" src="https://github.com/winglang/wing/assets/3268013/f4f966c5-a917-4e27-9754-ac69923e01e6">

Also the new patterns/examples page I'm working on, maybe we can also add a filter by platform too, this will also help with discoverability.

I will add a page into the docs with a table/list of platforms as a first step.

Reply:
Added a new here @eladb https://github.com/winglang/wing/pull/6859, let me know what you think